src/FakeItEasy/RootModule.cs

```diff
@@ -188,7 +188,7 @@ public bool TryCreateFakeObject(DummyCreationSession session, Type typeOfFake, D
         private class ArgumentConstraintManagerFactory
             : IArgumentConstraintManagerFactory

```

Comment:
Of course I meant `Negatable`. I shouldn't review code and what confusing TV at the same time. :blush: